client/src/test/scala/drt/client/components/BigSummaryBoxTests.scala

```diff
@@ -367,7 +367,8 @@ object ApiFlightGenerator {
       rawIATA = iataFlightCode,
       Origin = Origin,
       PcpTime = if (PcpTime != 0) PcpTime else SDate.parse(SchDT).millisSinceEpoch,
-      SchDT = SchDT
+      SchDT = SchDT,
+      Scheduled = if (SchDT != "") SDate.parse(SchDT).millisSinceEpoch else 0L
```

Comment:
SDate.parse should probably return an Option - I've encountered this issue a few times. Possibly one for later though.